cloudinit/handlers/jinja_template.py

```diff
@@ -97,7 +101,9 @@ def render_jinja_payload_from_file(
 def render_jinja_payload(payload, payload_fn, instance_data, debug=False):
     instance_jinja_vars = convert_jinja_instance_data(
         instance_data,
-        decode_paths=instance_data.get('base64-encoded-keys', []))
+        decode_paths=instance_data.get('base64-encoded-keys', []),
+        include_key_aliases=True
```

Comment:
Is `include_key_aliases` being used in this function?

Reply:
ahh no longer in this iteration.

Reply:
The `instance_jinja_vars` output relies on the included aliases in the return dict to pass into render_string(). Without the added aliases in the instance_jinja_vars dict we wouldn't get proper alias path substitutions.

Reply:
Ah right, I misread that.